goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -417,26 +439,49 @@ check_if_bmcs_are_reachable
 # for each BMC
 for i in "${NCN_BMCS[@]}"
 do 
-  
+
+  if [[ "$VENDOR" = *"Marvell"* ]] \
+    || [[ "$VENDOR" = "HP"* ]] \
+    || [[ "$VENDOR" = "Hewlett"* ]]; then
+    if [[ "$i" == *"ncn-m001"* ]]; then
+      # login locally
+      ilorest --nologo login 1>/dev/null
+    else
+      # login to the remote bmc 
+      ilorest --nologo login "$i" 1>/dev/null
+    fi
+
+  fi
+
   check_firmware_version "$i" \
     "$BMC_USERNAME" \
     "$IPMI_PASSWORD"
 
   check_bios_version "$i" \
     "$BMC_USERNAME" \
     "$IPMI_PASSWORD"
-    
+
+  if [[ "$VENDOR" = *"Marvell"* ]] \
+    || [[ "$VENDOR" = "HP"* ]] \
+    || [[ "$VENDOR" = "Hewlett"* ]]; then
+    ilorest --nologo logout 1>/dev/null
+  fi
+
 done
 
 if [[ $BASELINE = Y ]]; then
   /root/bin/bios-baseline.sh --check
   exit $?
```

Comment:
```suggestion
```

Reply:
This is actually separate from my script. I don't want to roll changes to that into this PR.

Reply:
Fixed after talking with you